### PR TITLE
Add .travis.yml for testing

### LINKED ISSUE #6: add a Travis-CI build
volunteer to add a `.travis.yml`, for PR validation?

### PR DESCRIPTION
**Pull request by jan0sch**
* add configuration for travis ci
* fixes #6

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+language: scala
+scala:
+  - 2.13.1
+jdk:
+  - openjdk8
+  - openjdk11
+sudo: false
```
Comment (SethTisue) [inline .travis.yml]: I believe this line can be omitted, it doesn't do anything these days, on current Travis-CI

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+script:
+  - sbt clean test
```
Comment (SethTisue) [inline .travis.yml]: `clean` isn't necessary, we are beginning from a clean state

also, the `scala:` declaration above won't take effect unless you do this:

`sbt ++$TRAVIS_SCALA_VERSION test`

File: .travis.yml (hunk 1 above)
Comment (jan0sch) [reply .travis.yml]: Removed.

File: .travis.yml (hunk 2 above)
Comment (jan0sch) [reply .travis.yml]: Removed. :-)

**Comment by jan0sch** [comment]
The build on my fork can be seen here: https://travis-ci.org/wegtam/spotted-leopards

**Comment by mpilquist** [comment]
Just enabled Travis on this repo

**Pull request merged**